### DISCUSSION TITLE
Erro no mapos 4.47

### DISCUSSION THREAD

**Discussion by ClaudioRSB13** [RamonSilva20/mapos#2555] [General]
Boa noite pessoal,
Estava com o mapos 39  ou era 41 na kinghost porém fui atualizar pelo próprio mapos e deu pau. Tentei reinstalar e conectar de novo no banco e crash.
Então para resolver ou descobrir o que está acontecendo fui pra instalação local, tudo ok com as instalações, sem erros testei em 4.45, 4.46 e 4.47 todas funcionam perfeitamente, mas quando vou restaurar meu banco eles crasham em todas essas versões, alguém tem alguma sugestão?

![Captura de tela 2024-10-01 194927](https://github.com/user-attachments/assets/3185e93b-e1d3-4c77-8b51-43db97409c51)

![Captura de tela 2024-10-01 194827](https://github.com/user-attachments/assets/ad92f661-c066-45cb-923b-479d84658001)

Atualizando nas versões depois mapos-4.44.0 até mapos-4.47.0 crasham após restaurar banco de dados, da 4.44.0  para trás funciona tudo ok após a restauração.

**Comment by allanduraes** [comment]
Olá, estou enfrentando o mesmo erro e suspeito que possa ser algo relacionado ao banco de dados, pois atualizei da versão 4.45 para a 4.47. Isso pode ter sido a causa do problema. Ainda não descobri como solucionar, mas se você conseguir, por favor, me avise.

**Comment by Wallidantas** [comment]
Eu uso a versão 4.40 já faz um bom tempo, ai fui atualizar, quando jogo meu banco de dados da a mesma coisa.. ninguém conseguiu resolver?

**Comment by leandr0ferreir4** [comment]
Eu tive um problema semelhante após uma atualização, sempre que eu fazia login, sempre mostrava um erro semelhante e pra ir pra sessão de ordem de serviços eu tinha que abrir o endereço manualmente, por exemplo (https://dominio.com/mapos/index.php/os/visualizar/438)  meu problema, principalmente, era pra abrir a tela inicial do mapOS e outras telas, como cobranças e lançamentos, a minha instalação está hospedada na Hostinger, como eu sempre faço a atualização manualmente e percebi que o comando "composer install --no-dev" dava falha para atualizar algumas dependências, tentei resolver, mas sem sucesso. No final, resolvi o problema criando uma nova instalação, inclusive com um novo banco, fiz o backup do banco da instalação antiga (https://dominio.com/mapos/index.php/mapos/backup) e executei a nova instalação, acessei o novo banco de dados no phpmyadmin, importei o banco e copiei os dados das pastas "Anexos", "Arquivos" e etc para a pasta da nova instalação, com isto resolvi o problema.
